o3de/o3de-extras
# MP Template Stability Update May 2024

## What does this PR do?

Stability updates and fixes to the o3de-extras MPTemplate project.
- Removing corrupted icon data from Resources folder.
- Removing depreciated spawner code, updating lists and template file to reflect changes.
- Updating Demo level to support new simple spawning component.
- Tested working starting from scratch using Create Project window.
 - (I changed the Template folder body to dev branch o3de-extras template version)

Note: This update does not change the default Template button on the Project Manager. That is a separate issue here: https://github.com/o3de/o3de/issues/17989

## How was this PR tested?

1. Go to Create Project window in Project Manager.
2. Download the MPTemplate via the template buttons.
3. Navigate to your Template download destination. Delete the files in the MPTemplate folder.
4. Copy content from this versions MPTemplate into your local Template folder.
5. Create new project with that edited Template Folder.
6. The project should have a pink network preview image.
7. Build.
8. Build should succeed, may have a warning anyways.
9. Launch Editor even if the build has a warning icon.
10. Let the Asset Manager load.
11. Load Demo.level
12. Play from Editor after the boxes become visible due to asset processing completing.
13. You should be able to run around as BURT and shoot and crouch and things.
